_episodes/14-writing-functions.md

```diff
@@ -77,17 +77,14 @@ print_date(1871, 3, 19)
 ~~~
 {: .output}
 
-*   Via [Twitter](https://twitter.com/minisciencegirl/status/693486088963272705):
```

Comment:
It doesn't really make sense to me - I actually thought it had been left in by error as the twitter discussion doesn't really reflect the statement it's linked to and is hard to follow.  I don't think calling arguments "ingredients" helps the understanding, in fact, ingredients imply specificity whereas arguments allow for flexibility.   There are references in an earlier lesson to the correct nomenclature (02-variables) so a deviation at this point could create confusion.